aws/awscloud/awscloud.go

```diff
@@ -37,5 +43,27 @@ var AWS = wire.NewSet(
 // AWS set, does not include credentials. Individual services may require
 // additional configuration.
 var Services = wire.NewSet(
+	// gocloud.dev/blob
```

Comment:
Will it be a better idea to split these into their driver packages? For example, in `s3blob` package:
```go
var Set = wire.NewSet(
    s3blob.Options{},
    s3blob.URLOpener{})
```

Reply:
Done.